testnets/permtestnet/assetslist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "permtestnet",
+    "assets": [
+      {
+        "description": "Perm A Permissioned Blockchain",
+        "denom_units": [
+          {
+            "denom": "ufee",
+            "exponent": 0
+          },
+          {
+            "denom": "uperm",
```

Comment:
I'm guessing this should be 'perm' and not 'uperm'. Typically the u- prefix is added to act as a 'micro-' prefix, hence taking place of the base unit where the token has 6 exponent. If this were just 'perm', and not 'uperm', it would make the 'display' unit make sense.